internal/assessor/schema_test_helpers_test.go

```
@@ -0,0 +1,108 @@
+}
+
+func schemaPropertiesForTest(t *testing.T, schema map[string]any) map[string]any {
+	t.Helper()
+	props, ok := schema["properties"].(map[string]any)
+	if !ok {
+		t.Fatalf("schema has no properties: %#v", schema)
+	}
+	return props
+}
+
+func itemSchemaForTest(t *testing.T, schema any) map[string]any {
```

golangci-lint (2.12.2)

[error] func itemSchemaForTest is unused

(unused)